docker/README.md

```diff
@@ -1,5 +1,9 @@
 ![bentoml-docker](tools/bentoml-docker.png)
 
+BentoML enables users to create a machine learning powered prediction service in minutes and bridges the gap between data science and DevOps.
```

Comment:
@timliubentoml this is actually a developer facing API explaining how to add new docker base image, the user facing one is a template file located at `docker/templates/docs/README.md.j2`: https://github.com/bentoml/BentoML/blob/main/docker/templates/docs/README.md.j2